test/helpers/foundry/helpers.ts

```diff
@@ -298,19 +305,29 @@ async function extractFromTar(
             const passThrough = new Minipass({ async: true });
             passThrough.pipe(hash);
             passThrough.on('end', () => {
+              const absolutePath = entry.absolute;
+              if (!absolutePath) {
+                throw new Error('Missing absolute path for entry');
+              }
```

Comment:
Can move this, and the identical `absolutePath` check below, above the `if` and run it only once?

Reply:
nice catch :heart_eyes: adding the changes! thank you!!